internal/codegen/tpl_stencil.go

```diff
@@ -119,23 +120,43 @@ func (s *TplStencil) Args() map[string]interface{} {
 //
 //	{{ stencil.ReadFile "myfile.txt" }}
 func (s *TplStencil) ReadFile(name string) (string, error) {
-	cwd, err := os.Getwd()
+	f, ok := s.exists(name)
+	if !ok {
+		return "", errors.Errorf("file %q does not exist", name)
+	}
+
+	b, err := io.ReadAll(f)
 	if err != nil {
 		return "", err
 	}
 
-	f, err := osfs.New(cwd).Open(name)
+	return string(b), nil
+}
+
+// Exists returns true if the file exists in the current directory
+//
+//	{{- if stencil.Exists "myfile.txt" }}
+//	{{ stencil.ReadFile "myfile.txt" }}
+//	{{- end }}
+func (s *TplStencil) Exists(name string) bool {
+	f, ok := s.exists(name)
+	f.Close() // close the file handle, since we don't need it
```

Comment:
f.Close() if ok?

Reply:
yeah, i fixed this locally and then got pulled into a meeting. Thank you!